### ISSUE TITLE
[E4] Interfaz gráfica Fire-Emblem-With-Gui.zip

### ISSUE THREAD

**Issue by camibennett** [franciscogazitua/IIC2113-2024#107]
Hola!
Estoy probando la interfaz gráfica con el proyecto Fire-Emblem-With-Gui.zip. Cuando corro el proyecto se me abre la ventana y me deja escoger las units de cada jugador, pero cuando inicia el combate me salen otras units (como si estuviera hardcodeado). Siempre me salen estas units independiente de cuales escoja:
<img width="1440" alt="Captura de pantalla 2024-06-17 a la(s) 11 20 15 a  m" src="https://github.com/franciscogazitua/IIC2113-2024/assets/89314958/e88d4b8e-68af-4cd3-a551-3aea806e0fcc">

Quedo atenta :)

**Comment by camibennett** [comment]
Acabo de darme cuenta que efectivamente esta hardcodeado. Gracias de todas formas!

**Comment by RodrigoToroIcarte** [comment]
Sí, está harcodeado. La idea es que ustedes lo hagan funcionar :)
